powershell/resources/psruntime/BuildTime/Models/PsProxyOutputs.cs

```diff
@@ -216,6 +216,25 @@ public string GetProcessCustomAttributesAtRuntime()
 {Indent}{Indent}[Microsoft.Message.ClientRuntime.MessageAttributeHelper]::ProcessPreviewMessageAttributesAtRuntime($cmdInfo, $MyInvocation, $parameterSet, $PSCmdlet)";
         }
 
+        private string GetLoginVerification()
+        {
+            if (!VariantGroup.IsInternal && IsAzure)
+            {
+                return $@"
+{Indent}{Indent}$context = Get-AzContext
+{Indent}{Indent}if (-not $context -or -not $context.Account) {{
+{Indent}{Indent}{Indent}Write-Error ""No Azure login detected. Please run 'Connect-AzAccount' to log in.""
+{Indent}{Indent}{Indent}exit
+{Indent}{Indent}}}
+{Indent}{Indent}if (-not $context.Subscription -or -not $context.Subscription.Id) {{
```

Comment:
Please remove the check for subscription, since in some cases, there are not any subscriptions.